zanata-war/src/main/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -293,6 +293,15 @@
     <to last="true">/webtrans/Application.seam$1</to>
   </rule>
 
+  <!-- zanata-assets resource -->
+  <!-- JSF serves zanata-assets with suffix of .seam,
+      this is to make sure any reference to zanata-assets without .seam can access to the resource.
+      e.g. /assets/style.css are redirect to /assets/style.css.seam -->
+  <rule>
+    <from casesensitive="true">^/javax.faces.resource/assets/(.*)(?&lt;!.seam)$</from>
+    <to last="true">/javax.faces.resource/assets/$1.seam</to>
```

Comment:
If we make this a little more general (not just `.../assets/`) we could use it for the pre-packaged WebJars from webjars.org too, like the ones for jQuery and Angular (or commonmark.js).  

I think we could just map from /webjars/* to `/javax.faces.resource/webjars/$1.seam` and we would also need to modify the zanata-assets webjar to be consistent with webjars.org, ie `META-INF/resources/webjars/zanata-assets/{version}`